server-csharp/Directory.Build.targets

```diff
@@ -0,0 +1,4 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Only needed when referencing local dependencies as projects. For published packages, these are imported automatically. -->
+  <Import Project="../../../SpacetimeDB/crates/bindings-csharp/Runtime/build/SpacetimeDB.Runtime.targets" />
+</Project>
```

Comment:
We should do a future PR to set this to point to the released version of SpacetimeDB bindings once it's released.